tests/lib/test_yaml_ext.py

```diff
@@ -233,6 +233,22 @@ def test_c_emitter(data_filename, canonical_filename, verbose=False):
 test_c_emitter.unittest = ['.data', '.canonical']
 test_c_emitter.skip = ['.skip-ext']
 
+def test_large_file(verbose=False):
+    SIZE_LINE = 24
+    SIZE_ITERATION = 0
+    SIZE_FILE = 31
+    if sys.maxsize <= 2**32:
+        return
+    if not os.environ.get('PYYAML_TEST_LARGE_FILE'):
```

Comment:
Just to kinda future-proof this under the assumption there are other tests that may need skipping in the future, how about  `os.environ.get('PYYAML_TEST_GROUP', '') == 'all'`- then we can just always set it to `all` in CI and anything else that checks that or something similar later will continue working...